Sources/GRPC/ClientConnection.swift

```diff
@@ -467,6 +468,9 @@ extension ClientConnection {
   public struct Configuration {
     /// The target to connect to.
     public var target: ConnectionTarget
+    
+    ///The HTTP/2 flow control target window size.
+    public var targetWindowSize: Int
```

Comment:
nit: can you name this `httpTargetWindowSize` and move it down to where `httpProtocol` is?